atest/testdata/parsing/same_setting_multiple_times.robot

```diff
@@ -28,11 +28,18 @@ Use Defaults
     Sleep    0.1s
 
 Test Settings
-    [Documentation]    T1
+    [Documentation]    FAIL Several failures occurred:\n\n
+    ...  1) Setting 'Documentation' is allowed only once. Only the first value is used.\n\n
+    ...  2) Setting 'Tags' is allowed only once. Only the first value is used.\n\n
+    ...  3) Setting 'Setup' is allowed only once. Only the first value is used.\n\n
+    ...  4) Setting 'Teardown' is allowed only once. Only the first value is used.\n\n
+    ...  5) Setting 'Teardown' is allowed only once. Only the first value is used.\n\n
+    ...  6) Setting 'Template' is allowed only once. Only the first value is used.\n\n
+    ...  7) Setting 'Timeout' is allowed only once. Only the first value is used.\n\n
+    ...  8) Setting 'Tags' is allowed only once. Only the first value is used.
```

Comment:
Now that these errors prevent the whole execution, the "Only the first value is used." part in the error message is unnecessary and actually pretty confusing.

Reply:
This will by the way be different with suite settings, they will still be only logged to console but do not cause runtime error